.github/workflows/main.yml

```diff
@@ -150,6 +154,12 @@ jobs:
           SA_PASSWORD: JdMsKZPBBA8kVFXVrj8d
         ports:
           - 1433:1433
+        options: >-
```

Comment:
Does this fix #17 ?

Reply:
I think it does. I haven't seen that issue again after this change, but then again it didn't occur every time. But I think I'll close #17 when this is merged and if it occurs again we'll open a new issue.